src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -267,7 +269,7 @@ private void handleKeys(boolean isForwardKey) {
     }
 
     private void scrollandShowPanel(int selectedPanelIndex, int numOfPanels) {
-        ui.getMenuControl().scrollTo(selectedPanelIndex, numOfPanels);
+        ui.scrollTo(selectedPanelIndex, numOfPanels);
```

Comment:
What I meant was the non-camel-cased name, but yup. :stuck_out_tongue: Those methods do the same thing.